positive-reputation-py/src/findings.py

```diff
@@ -23,6 +26,7 @@ def positive_reputation(address: str, chain_id: int) -> Finding:
             'alert_id': 'POSITIVE-REPUTATION-1',
             'type': FindingType.Info,
             'severity': FindingSeverity.Info,
-            'labels': labels
+            'labels': labels,
+            'metadata': {"anomaly_score": calculate_alert_rate(chain_id, BOT_ID, "POSITIVE-REPUTATION-1", ScanCountType.TRANSFER_COUNT)}
```

Comment:
Could you use `ScanCountType.TX_COUNT` for the denominator?
https://github.com/forta-network/bot-alert-rate/blob/main/python/bot_alert_rate/calculate_alert_rate.py#L20